8ball/chain.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "8ball",
+  "status": "live",
+  "website": "https://8ball.info/",
+  "network_type": "mainnet",
+  "pretty_name": "8ball",
+  "chain_id": "eightball-1",
+  "bech32_prefix": "8ball",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Registered